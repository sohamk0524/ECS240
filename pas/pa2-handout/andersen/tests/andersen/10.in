c Test case from the lecture slide 08
p 7 8
s 0 3 -1 6
s 0 5 -1 3
s 0 7 -1 1
s 0 1 0 4
s 0 2 0 1
s 0 4 1 5
s 1 5 0 2
s 1 7 0 5
