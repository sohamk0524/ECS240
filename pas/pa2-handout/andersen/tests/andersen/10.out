pt 1 3
pt 1 6
pt 2 3
pt 2 6
pt 3 3
pt 3 6
pt 4 3
pt 4 6
pt 5 3
pt 7 1
